configs/dask_cluster/dask_operator/catlas-hybrid-cluster.yml

```diff
@@ -41,6 +41,9 @@ spec:
             - mountPath: /home/jovyan/dask-worker-space
               name: shared-scratch
               subPath: catlas/dask-worker-space
+            - mountPath: /home/jovyan/.pmgrc.yaml
+              name: workspace-inference
```

Comment:
I dont think this will work? The pmgrc.yaml is setup for use with MPRester from pymatgen?

Reply:
Looks like the best route might be to have people store their api keys as github secrets: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

Reply:
just to be clear, these would not be github secrets they would be in kubernetes. Regardless, that feels just as clunky as the current solution? We could have a settings.py file where people add there MP_API key and we can imported where needed. i.e. `from catlas.settings import MP_API_KEY`. At least that would keep in central so you wouldnt have to make the change to every single worker config yml

Reply:
and try to read from environment variables first so we can export it to the workers for github actions if needed?